lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -125,6 +129,73 @@ def is_email(identifier):
     return True
 
 
+class TestAdminAccessCoachDashboard(SharedModuleStoreTestCase, LoginEnrollmentTestCase):
```

Comment:
if you look at the current code, I created a [class](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/ccx/tests/utils.py#L26) in `ccx.tests.util` that has many of the following methods: you can inherit from that one instead of building/copying your own.